### ISSUE TITLE
Set of tuples type gets flagged as an error

### ISSUE THREAD

**Issue by rhacking** [PyCQA/pylint#1738]
### Current behavior
```python
from typing import Tuple, Set
test = Set[Tuple[str, str]]
```

Results in `E1126:Sequence index is not an int, slice, or instance with __index__`

### Expected behavior
This should not be flagged as an error

### pylint --version output
```
No config file found, using default configuration
__main__.py 1.7.4,
astroid 1.5.3
Python 3.6.3 (v3.6.3:2c5fed8, Oct  3 2017, 18:11:49) [MSC v.1900 64 bit (AMD64)]
```

**Comment by rhacking** [comment]
Nevermind, incorrect setup. It was actually using 1.6.4, it works fine on 1.7.4
